src/test/java/space/obminyashka/items_exchange/service/MailServiceTest.java

```diff
@@ -54,4 +54,20 @@ void sendMail_shouldPassTheFlow() throws IOException {
                         capturedRequest.getBody().contains(emailTo)));
         assertNotNull(confirmationCode);
     }
+
+    @Test
+    void sendMailForResetPassword_shouldPassTheFlow() throws IOException {
```

Comment:
The test above is almost equals to the current one. Would you mind trying to combine them into a parametrized test?

Reply:
combined into a parametrized test